Exdi/exdigdbsrvsample/GdbSrvControllerLib/systemregisters.xml

```diff
@@ -0,0 +1,172 @@
+<SystemRegisterMap>
+	<SystemRegisters Architecture = "ARM64" >
+	    <!-- system core registers -->
+		<register Name ="SPSR_EL1"   AccessCode = "3, 0,  4, 0, 0" />
+        <register Name ="SPSR_EL2"   AccessCode = "3, 4,  4, 0, 0" />
+        <register Name ="SPSR_EL3"   AccessCode = "3, 6,  4, 0, 0" />
+        <register Name ="ELR_EL1"    AccessCode = "3, 0,  4, 0, 1" />
+        <register Name ="ELR_EL2"    AccessCode = "3, 4,  4, 0, 1" />
+        <register Name ="ELR_EL3"    AccessCode = "3, 6,  4, 0, 1" />
+        <register Name ="SP_EL0"     AccessCode = "3, 0,  4, 1, 0" />
+        <register Name ="SP_EL1"     AccessCode = "3, 4,  4, 1, 0" />
+        <register Name ="SP_EL2"     AccessCode = "3, 6,  4, 1, 0" />
+        <register Name ="FPSR"       AccessCode = "3, 3,  4, 4, 1" />
+        <register Name ="FPCR"       AccessCode = "3, 3,  4, 4, 0" />
+        <register Name ="SPSR_ABT"   AccessCode = "3, 4,  4, 3, 1" />
+        <register Name ="IFSR32_EL2" AccessCode = "3, 4,  5, 0, 1" />
+
+        <!-- ID and feature registers -->
+        <register Name ="VPIDR_EL2"        AccessCode = "3, 4,  0, 0, 0" />
+        <register Name ="ARM64_REVIDR_EL1" AccessCode = "3, 0,  0, 0, 6" />
+        <register Name ="CTR_EL0"          AccessCode = "3, 3,  0, 0, 1" />
+        <register Name ="ARM64_VMPIDR_EL2" AccessCode = "3, 4,  0, 0, 5" />
+        <register Name ="ID_AA64PFR1_EL1"  AccessCode = "3, 0,  0, 4, 1" />
+		<register Name ="ID_AA64DFR0_EL1"  AccessCode = "3, 0,  0, 5, 0" />

```

Comment:
It looks like there is a mix of spaces and tabs in this file. Would be good to make it uniform.